io.github.babaissarkar.ssplot.json

```diff
@@ -0,0 +1,43 @@
+{
+    "app-id": "io.github.babaissarkar.ssplot",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions" : [ "org.freedesktop.Sdk.Extension.openjdk17" ],
+    "command": "ssplot-run",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--filesystem=xdg-documents:rw"
+    ],
+    "modules": [
+        {
+            "name" : "openjdk",
+            "buildsystem" : "simple",
+            "build-commands" : [ "/usr/lib/sdk/openjdk17/install.sh" ]
+        },
+        {
+            "name": "SSPlot",
+            "buildsystem": "simple",
+            "builddir": true,
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/babaissarkar/ssplot/archive/refs/tags/v2.2.2.tar.gz",
+                    "sha256": "3582d04d8d8be2b71ef5104a0d888f505e54b7df3c155647babd3ab6aabde247"
+                },
+                {
+                    "type": "file",
+                    "path": "ssplot-run"
+                }
+            ],
+            "build-commands": [
+                 "desktop-file-edit --set-icon=$FLATPAK_ID SSPlot.AppDir/ssplot.desktop",
+                 "install -Dm644 SSPlot.AppDir/ssplot.png -t $FLATPAK_DEST/share/icons/hicolor/128x128/apps/io.github.babaissarkar.ssplot.png",
+                 "install -Dm644 $FLATPAK_ID.metainfo.xml -t $FLATPAK_DEST/share/metainfo",
+                 "install -Dm644 SSPlot.AppDir/ssplot.desktop -t $FLATPAK_DEST/share/applications/$FLATPAK_ID",
```

Comment:
```suggestion
                 "install -Dm644 SSPlot.AppDir/ssplot.desktop $FLATPAK_DEST/share/applications/${FLATPAK_ID}.desktop",
```

Reply:
sorry I messed up that one. This suggestion is correct.